api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -178,6 +179,11 @@ private void registerGetBookLog(Router router) {
     getBookFlow.handler(this::handleGetBookLog);
   }
 
+  private void registerDeleteBookLog(Router router) {
+    Route deleteBookLog = router.delete("/:school_id/:book_id");
```

Comment:
I would change the route to /schools/school_id/books/book_id. That way it is clear that we are querying a specific book. That way it is clear that we are deleting a book from a specific school. EX: /schools/5/books/3. We want to delete the 3rd book from the 5th school.